src/app/homepage/menu/menu.component.ts

```diff
@@ -181,8 +181,8 @@ export class MenuComponent implements OnInit {
       children: [
         { title: 'Introduction', path: '/openapi/introduction' },
         {
-          title: 'Types and Parameters',
-          path: '/openapi/types-and-parameters',
+          title: 'Model schema',
```

Comment:
I merged types, parameters and now schema into 1 page called Model schema. Should I revert it back to types and parameters?

Reply:
I think we should keep the title the same unless we're going to have a new major version release from NestJS itself to avoid confusion. So yes, can we keep it `Types and Parameters`?

Reply:
Understood. I will revert back to Types and Parameters